ipaserver/install/ca.py

```diff
@@ -425,7 +425,7 @@ def install_step_1(standalone, replica_config, options, custodia):
 
 def uninstall():
     ca_instance = cainstance.CAInstance(api.env.realm)
-    ca_instance.stop_tracking_certificates()
+    ca_instance.stop_tracking_certificates(False)
```

Comment:
[nit] could you write the whole kwarg `stop_certmonger=False` to give meaning to the bool, here on the caller side?

Reply:
Yeah, sorry for being lazy. Fixed.